examples/models/speech_recognition/TfDeepSpeech.py

```diff
@@ -0,0 +1,1137 @@
+from __future__ import absolute_import, division, print_function
+
+import os
+import sys
+import time
+
+import tensorflow as tf
+from tensorflow.python.tools import freeze_graph
+from tensorflow.python.client import device_lib
+from tensorflow.python.framework.ops import Tensor, Operation
+from tensorflow.contrib.framework.python.ops import audio_ops as contrib_audio
+
+import numpy as np
+from datetime import datetime
+from multiprocessing import cpu_count
+from multiprocessing.dummy import Pool
+import itertools
+import tempfile
+import base64
+
+from ds_ctcdecoder import ctc_beam_search_decoder_batch, ctc_beam_search_decoder, Scorer
+
+from rafiki.model import BaseModel, FixedKnob, IntegerKnob, FloatKnob, CategoricalKnob, \
+    dataset_utils, logger, test_model_class, InvalidModelParamsException
+
+from rafiki.constants import TaskType, ModelDependency
+from rafiki.utils.text import Alphabet
```

Comment:
I put it under `rafiki.utils` because the `AudioFilesDataset`'s `_load` method uses `Alphabet`. As such:

` df['transcript'] = df['transcript'].apply(partial(text_to_char_array, alphabet=Alphabet('examples/datasets/speech_recognition/alphabet.txt')))`

If that is not suitable, I can move this processing step into the `TfDeepSpeech` model or `utils` within `examples/models/speech_recognition`.

Reply:
Moved above code chunk to `TfDeepSpeech.py`. 
`Alphabet` moved to `examples/models/speech_recognition/utils/text.py`.

Reply:
So if it is decided that this is part of the dataset-loading utils, `Alphabet` would be an abstraction offered by Rafiki's dataset-loading utils and it should be put in https://github.com/nginyc/rafiki/blob/master/rafiki/model/dataset.py.